ioos_operations/pre-commits.md

```diff
@@ -0,0 +1,9 @@
+# pre-commit hooks
+
```

Comment:
To provide context, can we include some high level description of a pre-commit hook. From https://pre-commit.com/ I found:

> Git hook scripts are useful for identifying simple issues before submission to code review. We run our hooks on every commit to automatically point out issues in code such as missing semicolons, trailing whitespace, and debug statements. By pointing these issues out before code review, this allows a code reviewer to focus on the architecture of a change while not wasting time with trivial style nitpicks.

Reply:
This first paragraph explains git hooks but, pre-commit, are just git hooks that runs linters, so I think it is a ok to add that one to explain what happens under the hood. I also added the link to the docs and a TL;DR line below just in case.